qemu/tests/cfg/pci_bridge.cfg

```diff
@@ -1,19 +1,41 @@
 - pci_bridge: install setup image_copy unattended_install.cdrom
     virt_test_type = qemu
     login_timeout = 360
+    type = pci_bridge
+    start_vm = no
+    no virtio_scsi
```

Comment:
I think this case only work with x86.  Need add this restriction.

`no virtio_scsi`  here, and `only virtio_blk` in L27.  Can it work with ide ?

Reply:
Updated, Thanks!